### PR TITLE
added highlight support in Nebula grid with custom renderer

### PR DESCRIPTION
**Pull request by theanuradha**
image

### CODE REVIEW

**Comment by theanuradha** [comment]
@IanMayo  I wrote my own **FilterTextCellRenderer**

**Comment by IanMayo** [comment]
Hi there, have left some feedback in issue #1885

File: org.mwc.cmap.NarrativeViewer/src/org/mwc/cmap/NarrativeViewer/FilterTextCellRenderer.java [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+      textLayout.draw(gc, getBounds().x + x, getBounds().y + textTopMargin
+          + topMargin);
+      textLayout.dispose();
+      textLayout = null;
+    }
+
+    if (item.getParent().getLinesVisible())
+    {
+      if (isCellSelected())
+      {
```
Comment (IanMayo) [inline org.mwc.cmap.NarrativeViewer/src/org/mwc/cmap/NarrativeViewer/FilterTextCellRenderer.java]: @theanuradha - can we discuss this next question?

Comment (theanuradha) [inline org.mwc.cmap.NarrativeViewer/src/org/mwc/cmap/NarrativeViewer/FilterTextCellRenderer.java]: this print code copy from DefaultCellRenderer so wha tthey do here is check cell selection enable and print shadow

Comment (theanuradha) [inline org.mwc.cmap.NarrativeViewer/src/org/mwc/cmap/NarrativeViewer/FilterTextCellRenderer.java]: [code extend DefaultCellRenderer ]

Comment (IanMayo) [inline org.mwc.cmap.NarrativeViewer/src/org/mwc/cmap/NarrativeViewer/FilterTextCellRenderer.java]: So we don't need it to be user definable?

Comment (theanuradha) [inline org.mwc.cmap.NarrativeViewer/src/org/mwc/cmap/NarrativeViewer/FilterTextCellRenderer.java]: no it is comment from eclipse source [DefaultCellRenderer] they maybe had a thought on " user definable" but never added .

Comment (IanMayo) [inline org.mwc.cmap.NarrativeViewer/src/org/mwc/cmap/NarrativeViewer/FilterTextCellRenderer.java]: aah, ok.  It's not ours.

**Pull request merged**